polynote-frontend/style/styles.less

```diff
@@ -2512,4 +2513,6 @@ button {
   }
 }
 
-
+.monaco-editor {
+  position: absolute !important;  // so the context menu can float over the editor area.
```

Comment:
What position is it without this? Doesn't this cause other problems?

Reply:
Normally it's `relative`, but since we put it inside our own div already it doesn't change much. I do notice a slight overflow on the right so I'll fix that. 

![Screen Shot 2019-07-08 at 10 17 50 AM](https://user-images.githubusercontent.com/5430417/60829464-cb8a7e80-a169-11e9-8f03-a0bd0aedcef2.png)

As far as I could tell by poking around it didn't cause other issues. 

Definitely open to alternatives here, I basically found this one through trial and error...

Reply:
Ok, fixed the display issue by putting the editor in another div